README.md

```diff
@@ -6,5 +6,6 @@ Protocol buffer files for TiKV
 + Write your own protocol file in proto folder.
 + Run `make` to generate go and rust code. 
     We generate all go codes in pkg folder and rust in src folder.
++ Run `mvn clean install` to generate and install Java dependencies locally
```

Comment:
this is included in tikv-client as submodules. maven install is just for convenience of users who wants to use it this way.